roachpb/metadata.go

```diff
@@ -107,24 +106,18 @@ func (a Attributes) SortedString() string {
 }
 
 // ContainsKey returns whether this RangeDescriptor contains the specified key.
-// TODO(tschottdorf): RKey.
```

Comment:
I think this TODO refers to the fact that this method should be changed to take an `RKey` instead of a `[]byte` argument, so it is still open.